jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "."
```

Comment:
Note: add comment explaining that this here so we can import top level folders without all the ../../../ dancing

Reply:
can't add comments in json file lol nvm